src/lib/dialog/dialog-content-directives.ts

```diff
@@ -79,6 +80,10 @@ export class MatDialogClose implements OnInit, OnChanges {
   },
 })
 export class MatDialogTitle implements OnInit {
+  /**
+   * Element ID of the dialog title HTML element. If no ID is specified,
+   * one is being generated automatically on instantiation.
```

Comment:
"one is being generated automatically on instantiation" ->
"one is generated automatically"